### PR TITLE
Remove tools moved to base image

### PR DESCRIPTION
**Pull request by surajssd**
Following components were moved to base image:

- Ansible galaxy
- vscode
- Azure Developer CLI
- Bicep

This commit removes the installation of these
components from the tools image.

---

Should be merged after the base image is cached with changes in https://github.com/Azure/CloudShell/pull/482 and the tools image has the tag of the new base.

### CODE REVIEW

**Comment by surajssd** [comment]
Docker image sizes with this change:

```console
REPOSITORY                        TAG                                                                 IMAGE ID       CREATED             SIZE
quay.io/surajd/cloudshell-tools   suraj-reduce-tools-image-contents-53cfc40-2024-08-Aug-14-13-32-50   c336f3f4a1ac   About an hour ago   6.94GB
quay.io/surajd/cloudshell         suraj-reduce-tools-image-contents-53cfc40-2024-08-Aug-14-13-32-50   a8c78b8b7f00   About an hour ago   6.05GB
```

**Comment by surajssd** [comment]
Tested locally end-to-end.

File: linux/tools.Dockerfile [hunk 1]
```diff
@@ -31,56 +31,9 @@ RUN az aks install-cli \
...
-    && wget -nv -q -O /usr/share/ansible/collections/ansible_collections/azure/azcollection/requirements.txt https://raw.githubusercontent.com/ansible-collections/azure/dev/requirements.txt \
-    && /opt/ansible/bin/python -m pip install -r /usr/share/ansible/collections/ansible_collections/azure/azcollection/requirements.txt
-
-# Powershell telemetry
-ENV POWERSHELL_DISTRIBUTION_CHANNEL=CloudShell \
-    # don't tell users to upgrade, they can't
-    POWERSHELL_UPDATECHECK=Off
-
-# Copy and run script to install Powershell modules and setup Powershell machine profile
-COPY ./linux/powershell/ powershell
```
Comment (priyaananthasankar) [inline linux/tools.Dockerfile:68]: Can merge everything in tools to base except "powershell" and az cli.

**Comment by surajssd** [comment]
With the latest changes the file sizes are as:

```bash
$ docker images
REPOSITORY                          TAG                                                         IMAGE ID       CREATED          SIZE
ghcr.io/surajssd/cloudshell/tools   suraj-reduce-tools-image-contents-c0483b0-20240821-225233   fec921d743f7   29 minutes ago   7.24GB
ghcr.io/surajssd/cloudshell/base    suraj-reduce-tools-image-contents-c0483b0-20240821-225233   8b7e8244234e   34 minutes ago   5.02GB
```

Tools and base has about 2.2GB difference.

**Comment by surajssd** [comment]
```bash
REPOSITORY                          TAG                                                         IMAGE ID       CREATED             SIZE
ghcr.io/surajssd/cloudshell/tools   suraj-reduce-tools-image-contents-d2aacba-20240829-194858   bc2abc431591   About an hour ago   7.24GB
ghcr.io/surajssd/cloudshell/base    suraj-reduce-tools-image-contents-d2aacba-20240829-194858   8a80d4c384cf   2 hours ago         5.02GB

```

**Comment by surajssd** [comment]
Closed in favor of https://github.com/Azure/CloudShell/pull/496

**Pull request closed without merging**